content/en/docs/concepts/services-networking/service.md

```diff
@@ -866,10 +866,7 @@ finding a Service: environment variables and DNS.
 When a Pod is run on a Node, the kubelet adds a set of environment variables
 for each active Service. It adds `{SVCNAME}_SERVICE_HOST` and `{SVCNAME}_SERVICE_PORT` variables,
 where the Service name is upper-cased and dashes are converted to underscores.
-It also supports variables
-(see [makeLinkVariables](https://github.com/kubernetes/kubernetes/blob/dd2d12f6dc0e654c15d5db57a5f9f6ba61192726/pkg/kubelet/envvars/envvars.go#L72))
-that are compatible with Docker Engine's
-"_[legacy container links](https://docs.docker.com/network/links/)_" feature.
+
```

Comment:
nit: Extra blank like isn't needed. Not a big deal since we don't do mdlint enforcement, but if you need to update for some other reason it would be nice to clean it up.

Reply:
Ok, @stmcginnis thanks for suggestion will do!